docs/user_guide.rst

```diff
@@ -1,2 +1,28 @@
 User Guide
 ==========
+
+.. contents:: :depth: 3
+
+Example
+-------
+
+This is an example for setting up FoxPuppet and running a simple check to see if the browser window is private::
+
+    from foxpuppet import FoxPuppet
+    from selenium.webdriver import Firefox
+
+
+    selenium = Firefox(
+        firefox_binary='path/to/firefox-bin'
+    )
+    foxpuppet = FoxPuppet(selenium)
+
+    window = foxpuppet.browser.open_window(private=True)
+    window.close()
+    foxpuppet.browser.close()
```

Comment:
I don't think we should advise this, as users would probably not be able to recover from closing the original browser window. The `selenium.quit` call will take care of closing any remaining windows.